src/plugins/intel_npu/src/backend/include/zero_executor.hpp

```diff
@@ -67,10 +68,23 @@ class ZeroExecutor final : public IExecutor {
 
     ze_graph_handle_t _graph = nullptr;
     ze_graph_properties_t _props{};
+
     std::unordered_map<std::string, ArgumentDescriptor> _inputs_desc_map;
     std::unordered_map<std::string, ArgumentDescriptor> _outputs_desc_map;
 
     std::array<std::shared_ptr<CommandQueue>, stage::COUNT> _command_queues;
+
+    template <typename T, std::enable_if_t<NotSupportArgumentProperties3(T), bool> = true>
```

Comment:
Let's be consistent with typename -- used only in in createGraph but not here

Reply:
sorry, but I do not understand. Could you explain a little bit more, please?

Reply:
done